spock-core/src/main/java/spock/lang/Retry.java

```diff
@@ -27,12 +27,23 @@
 /**
  * Retries the given feature if an exception occurs during execution.
  *
+ * <p>Retries can be applied to feature methods and spec classes. Applying it to
+ * a spec class has the same effect as applying it to each feature method that
+ * isn't already annotated with {@code Retry}.
+ *
+ * <p>{@code Retry} annotations are inherited, i.e. if declared on a superclass
+ * of a spec class, it will be applied to all feature methods of both classes
+ * unless the spec class declared its own annotation. If so, it is applied to
+ * all feature methods declared in that class, but not to any feature methods
+ * declared in superclasses.
```

Comment:
Add something to explain that `@Retry` also does not apply to inherited features